Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteScrollingCoordinatorProxyIOS.mm

```diff
@@ -262,8 +300,12 @@
 
     if (value)
         m_uiState.addNodeWithActiveUserScroll(rootNode->scrollingNodeID());
-    else
+    else {
         m_uiState.removeNodeWithActiveUserScroll(rootNode->scrollingNodeID());
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+        reconcileConnectedNodes();
```

Comment:
"reconcile" is too vague. This function is called when a scroll starts, not for every scroll increment, so I'm not sure why fixing up the interaction layer bounds here makes sense?